src/components/Dissolution/CompleteAffidavit.vue

```diff
@@ -336,6 +337,11 @@ export default class Affidavit extends Mixins(CommonMixin, DocumentMixin, EnumMi
     this.affidavitConfirmed = this.getAffidavitStep.affidavitConfirmed
     this.hasValidUploadFile = !!this.uploadAffidavitDocKey
     this.hasAffidavitConfirmed = this.affidavitConfirmed
+  }
+
+  async mounted (): Promise<void> {
+    // wait for components to load/stabilize then update validation state in store
+    await this.$nextTick()
     this.updateAffidavitStepValidity()
```

Comment:
This is testing another theory. I might revert this later, dunno, shouldn't hurt like this in the long term if I don't.